src/components/editMeasure/measureDetails/measureMetadata/MeasureMetadata.tsx

```diff
@@ -87,15 +93,42 @@ export default function MeasureMetadata(props: MeasureMetadataProps) {
               </SubHeader>
             </div>
           )}
+          {measureMetadataType !== "Steward" && (
+            <>
+              <p style={{ fontFamily: "Rubik", fontSize: 32, fontWeight: 400 }}>
+                {measureMetadataType}
+              </p>
+
+              <div
+                style={{
+                  display: "flex",
+                  flexDirection: "row",
+                  justifyContent: "flex-end",
+                }}
+              >
+                <Typography
+                  style={{ fontSize: 14, fontWeight: 300, fontFamily: "Rubik" }}
+                >
+                  <span style={{ color: "#D92F2F", marginRight: 3 }}>*</span>
+                  Indicates required field
+                </Typography>
+              </div>
+
```

Comment:
there is a lot of inline CSS. I think we can create classes

Reply:
We use Tailwind classes to style our application. It is ideal to follow a single convention

Reply:
@adongare This form uses tailwind classes which have predefined values and don't have values for everything, and to add our own css to fit figma design from the jira card, we'll need to pass the styled component a function that returns an array. I asked Matt about this, he suggested using styesheets instead of tailwind and it makes things less complicated and easier to manage.

Reply:
@sb-cecilialiu I think @adongare is referring to excessive use of style attributes. They process slower than classes.
Anywhere where there's a style attribute, you can switch it to a className and reference it in a stylesheet

Reply:
@adongare @mcmcphillips Ok I see, thanks!

Reply:
@adongare @RohitKandimalla @mcmcphillips Ok I changed the styles to use className